sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

```diff
@@ -2692,6 +2700,372 @@ public Response<DynamicClassifyDocumentResultCollection> dynamicClassificationBa
             documents, projectName, deploymentName, options, context);
     }
 
+    // Abstractive Summarization
+    /**
+     * Returns a list of abstract summary for the provided list of {@link String document}.
+     *
+     * <p>This method is supported since service API version {@link TextAnalyticsServiceVersion#V2022_10_01_PREVIEW}.</p>
+     *
+     * This method will use the default language that can be set by using method
+     * {@link TextAnalyticsClientBuilder#defaultLanguage(String)}. If none is specified, service will use 'en' as
+     * the language.
+     *
+     * <p><strong>Code Sample</strong></p>
+     * <!-- src_embed Client.beginMultiLabelClassify#Iterable-String-String -->
+     * <pre>
+     * List&lt;String&gt; documents = new ArrayList&lt;&gt;&#40;&#41;;
+     * for &#40;int i = 0; i &lt; 3; i++&#41; &#123;
+     *     documents.add&#40;
+     *         &quot;I need a reservation for an indoor restaurant in China. Please don't stop the music.&quot;
+     *             + &quot; Play music and add it to my playlist&quot;&#41;;
+     * &#125;
+     * SyncPoller&lt;ClassifyDocumentOperationDetail, ClassifyDocumentPagedIterable&gt; syncPoller =
+     *     textAnalyticsClient.beginMultiLabelClassify&#40;documents, &quot;&#123;project_name&#125;&quot;, &quot;&#123;deployment_name&#125;&quot;&#41;;
+     * syncPoller.waitForCompletion&#40;&#41;;
+     * syncPoller.getFinalResult&#40;&#41;.forEach&#40;documentsResults -&gt; &#123;
+     *     System.out.printf&#40;&quot;Project name: %s, deployment name: %s.%n&quot;,
+     *         documentsResults.getProjectName&#40;&#41;, documentsResults.getDeploymentName&#40;&#41;&#41;;
+     *     for &#40;ClassifyDocumentResult documentResult : documentsResults&#41; &#123;
+     *         System.out.println&#40;&quot;Document ID: &quot; + documentResult.getId&#40;&#41;&#41;;
+     *         for &#40;ClassificationCategory classification : documentResult.getClassifications&#40;&#41;&#41; &#123;
+     *             System.out.printf&#40;&quot;&#92;tCategory: %s, confidence score: %f.%n&quot;,
+     *                 classification.getCategory&#40;&#41;, classification.getConfidenceScore&#40;&#41;&#41;;
+     *         &#125;
+     *     &#125;
+     * &#125;&#41;;
+     * </pre>
```

Comment:
seems like the wrong sample (edits: same for all the variants of this method)

Reply:
I forget to update the codesnap src_embed names. Thanks Krista.